pkgs/development/libraries/libcamera/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchgit
+, lib
+, meson
+, ninja
+, pkg-config
+, boost
+, gnutls
+, openssl
+, libevent
+, lttng-ust
+, libtiff
+, qt5
+, gst_all_1
+, graphviz
+, doxygen
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation {
+  pname = "libcamera";
+  version = "1.0.0";
+
+  src = fetchgit {
+    url = "git://linuxtv.org/libcamera.git";
+    rev = "040a6dcfbccaf1005da2c571769b1f89df1f07cf";
+    sha256 = "sha256-RCNKFCv3fNabroCUR5MtKbyDS20J7leU8KkE1U5qe1M=";
+  };
+
+  postPatch = ''
+    patchShebangs utils/
+  '';
+
+  buildInputs = [
+    # core requirements
+    python3Packages.pyyaml
+    python3Packages.ply
+    python3Packages.jinja2
+
```

Comment:
What about systemd for device hotplug enumeration?

Reply:
Hmm, it's not listed as a dependency on libcamera itself - I'm not sure how to get it to compile with it either?

Reply:
It is listed in readme:

> for device hotplug enumeration: [optional]
> 	libudev-dev

(udev is part of systemd)

Reply:
https://libcamera.org/getting-started.html this page lists the optional dependencies